tests/unit/components/loader/Loader.spec.jsx

```diff
@@ -0,0 +1,10 @@
+import { render, screen } from '@testing-library/react'
+import Loader from '~/components/loader/Loader'
+
+describe('Loader test', () => {
+  test('should be in the document', () => {
```

Comment:
I was interested to find out [when it is better to use `it` and when to use `test`](https://daily-dev-tips.com/posts/jest-test-vs-it/). Maybe you'll find it useful too.

Reply:
Thanks, I'll keep that in mind in the future.

Reply:
I can add to this conversation. In one project we must use one approach and code style. So 
```suggestion
  it ('should be in the document', () => {
```